test/built-ins/Temporal/PlainYearMonth/prototype/subtract/options-undefined.js

```diff
@@ -15,7 +15,6 @@ class CheckedAdd extends Temporal.Calendar {
   }
   dateAdd(date, duration, options, constructor) {
     this.called = true;
-    assert.notSameValue(options, undefined, "options not undefined");
```

Comment:
I think we still need to verify this on the 3rd dateAdd call, otherwise the test will no longer be testing what it claims to.